src/lightbox/LightboxActionSheet.js

```diff
@@ -1,6 +1,6 @@
 /* @flow strict-local */
 import type { Auth } from '../types';
-import downloadImage from './downloadImage';
```

Comment:
Agreed re: splitting the file rename from the rest.

I'd actually be happy to have the other two squashed together, because the new codepath is so small and is introduced for the sake of the new function. I agree I wouldn't label them `[nfc]`, for the reason you mention at the end.